libs/workorderapp/src/lib/config/gardenConfig.ts

```diff
@@ -34,10 +34,9 @@ export const useGardenConfig = (
     getSubgroupItems,
     customViews: {
       customItemView: GardenItem as any,
-      customHeaderView: GardenHeader as any,
     },
     visuals: {
-      rowHeight: 30,
+      rowHeight: 31,

```

Comment:
Could we do rowheight 30 for every garden? 31 feels weird unless its strictly necessary